docs/appendix/yaml.rst

```diff
@@ -188,8 +215,8 @@ included. You may include as many plotting groups as you like.
 For each plotting group, update the label and include the following information.
 Note: the labels need to be unique, but otherwise are not used.
 
-**type:** The model type. Options are: 'timeseries', 'taylor', 'spatial_bias',
-'spatial_overlay', 'boxplot'
+**type:** The model type. Options are: "timeseries", "taylor", "spatial_bias",
+"spatial_overlay", "spatial_bias_exceedance", and "boxplot"
```

Comment:
"spatial_bias_exceedance" only works when regulatory is true.

Reply:
Yes, I explain more details of these plots including that spatial_bias_exceedance only works when regulatory = True here: docs/background/supported_plots.rst 
Does this work or do you think we should explain these details in the yaml file description too?

Reply:
Actually I just added the note below too. Duplicity is probably useful here.

Reply:
I do not have other comments on the regulatory calculations. You have covered everything.